package.json

```diff
@@ -14,6 +14,7 @@
   "license": "Apache-2.0",
   "repository": "http://github.com/kruppel/check-engines",
   "dependencies": {
+    "cross-spawn": "^5.0.1",
```

Comment:
I'm a little hesitant to pull in a dependency for this, though it's tricky if the `spawn` option works for only Node versions >= 6. Beyond that, should the version range be so strict? Is there any reason to not `>=5.0.1`?